.github/workflows/test-pr.yml

```diff
@@ -328,7 +367,7 @@ jobs:
           set -euxo pipefail
           workspace=$(pwd)
           docker run --name k-profiling-tests-${GITHUB_SHA}                    \
-            --rm -it --detach                                                  \
+            --rm -it --detach --user root                                      \
```

Comment:
What changed between now and before that requires running as root in the container? Not an ideal scenario if we're publishing our tool but only works when used by 'root' -- primary concern for this change.